### DISCUSSION TITLE
load_model from AWS S3 using AWS EMR

### DISCUSSION THREAD

**Discussion by jffengineer** [moj-analytical-services/splink#2394] [Q&A] [answered]
How do I load my model from AWS S3 in AWS EMR?   load_model() appears to access local file system, even if I specify "s3://path_to_file/model.json".   Do I need to mount S3 on local nodes?   Thank you.

**Comment by RobinL** [comment, marked as answer]
You need to either download the JSON from S3 to local disk first and provide Splink the local file path or load from S3 into a python dictionary, and provide Splink with the python dict

**Reply by jffengineer** [reply]
Thank you Robin
